solr/solrj/src/java/org/apache/solr/client/solrj/request/QueryRequest.java

```diff
@@ -61,8 +61,8 @@ public String getPath() {
   // ---------------------------------------------------------------------------------
 
   @Override
-  protected QueryResponse createResponse(SolrClient client) {
-    return new QueryResponse(client);
+  protected QueryResponse createResponse(NamedList<Object> namedList) {
+    return new QueryResponse();
```

Comment:
I overlooked this spot; it uses client as a cache for caching the "DocumentObjectBinder" on the SolrClient.  That seems like a bit of a hack to me; should be cached somewhere, maybe even a static global or ThreadLocal.  I don't see it as bound to the client instance.

Reply:
I addressed this in another issue/PR that will merge first.